src/coreclr/tools/aot/ILCompiler.Compiler/IL/ILImporter.Scanner.cs

```diff
@@ -1070,7 +1070,7 @@ private void ImportStoreField(int token, bool isStatic)
             ImportFieldAccess(token, isStatic, isStatic ? "stsfld" : "stfld");
         }
 
-        private void ImportLoadString(int token)
+        private void ImportLoadString(int _)
```

Comment:
Because these methods are actually used as common interface for aot compiler and il-verifier as I understood it. I cannot change the signature because the call site needs to pass the value in il-verifier case.